metadata.json

```diff
@@ -9,7 +9,7 @@
   "issues_url": "https://github.com/jbeard6/jbeard-nfs/issues",
   "description": "This module installs an NFS client and server and configures the exports.",
   "dependencies": [
-    {"name":"jbeard/portmap","version_requirement":">= 0.1.6"},
+    {"name":"jbeard/portmap","version_requirement":">= 0.1.6 <2.0.0"},
```

Comment:
This doesn't look like the right line for this.  Did puppetlabs release a 2.x version of concat at some point and cause your puppet install to get confused?  Their current release is 1.2.3 from June 2, 2015.